canonical-web-and-design/canonicalwebteam.discourse
# Add preview feature

Redirect inactive engage page to discourse unless there is a `preview` flag.

## QA
For the sake of the QA I made /engage/it/deployment-azienda-manuale active = false. It will not affect prod in any way because the feature is not live yet.

1. Go to https://ubuntu-com-8761.demos.haus/engage/it/deployment-azienda-manuale you will be redirected to its discourse page.
2. Go to https://ubuntu-com-8761.demos.haus/engage/it/deployment-azienda-manuale?preview you will see the page
Press `ctrl + u` in browser. Use search to look for the meta tag robot. You will see it has the value `nofollow`.
3.  Go to https://ubuntu-com-8761.demos.haus/engage/it/redhat-openstack-confronto-manuale (an active page). Press `ctrl + u` in browser. Use search to look for the meta tag `robot`. It won't be there.

## Issue

Fixes: https://github.com/canonical-web-and-design/web-squad/issues/3404
